Source/WebCore/loader/FrameLoader.cpp

```diff
@@ -2230,6 +2230,7 @@ void FrameLoader::commitProvisionalLoad()
             frame->protectedView()->forceLayout();
 #else
         frame->protectedView()->forceLayout();
+        frame->protectedView()->updateCompositingLayersAfterLayout();
```

Comment:
Do we know why this is necessary?

Reply:
It would appear that it isn't, remnant of trying to track down failures.